packages/ignite-cli/src/templates/plugin/templates/thing.js.ejs.ejs

```diff
@@ -0,0 +1,3 @@
+// Replace with your own template "thing".
+
+<%= props.name %> // Pass in props from your command.
```

Comment:
A template file from which to generate "things" from. Components, models, whatever.